streams/ca.m3u

```diff
@@ -220,7 +220,7 @@ https://raw.githubusercontent.com/azgaresncf/strm2hls/main/streams/tvctklive.m3u
 https://tscamd.akamaized.net/hls/live/503340/TSCLive/master.m3u8
 #EXTINF:-1 tvg-id="Toronto360TV.ca",Toronto 360 TV (720p) [Not 24/7]
 http://cdn3.toronto360.tv:8081/toronto360/hd/playlist.m3u8
-#EXTINF:-1 tvg-id="",TSN The Ocho
+#EXTINF:-1 tvg-id="TSNTheOcho.ca",TSN The Ocho (1080p)
```

Comment:
channel with that name and tvg-id is not in database, please consider adding https://github.com/iptv-org/database/issues/new?assignees=&labels=channels%3Aadd&projects=&template=__channels_add.yml&title=Add%3A+

Reply:
the EPG is part of Samsung TV Plus. I made a PR yesterday to add it.

https://github.com/iptv-org/epg/pull/2414

Reply:
both EPG and iptv-org playlist rely on [https://github.com/iptv-org/database] repo